src/main/java/net/earthcomputer/clientcommands/command/KitCommand.java

```diff
@@ -0,0 +1,191 @@
+                                .suggests((ctx, builder) -> CommandSource.suggestMatching(kits.keySet(), builder))
+                                .executes(ctx -> edit(ctx.getSource(), StringArgumentType.getString(ctx, "name")))))
+                .then(literal("load")
+                        .then(argument("name", StringArgumentType.string())
+                                .suggests((ctx, builder) -> CommandSource.suggestMatching(kits.keySet(), builder))
+                                .then(literal("--override")
+                                        .executes(ctx -> load(ctx.getSource(), StringArgumentType.getString(ctx, "name"), true)))
+                                .executes(ctx -> load(ctx.getSource(), StringArgumentType.getString(ctx, "name"), false))))
+                .then(literal("list")
+                        .executes(ctx -> list(ctx.getSource()))));
+    }
+
+    private static int create(ServerCommandSource source, String name) throws CommandSyntaxException {
+        loadFile();
+
+        if (kits.get(name) != null) {
+            throw ALREADY_EXISTS_EXCEPTION.create(name);
+        }
+        kits.put(name, client.player.inventory.serialize(new ListTag()));
+        saveFile();
+        sendFeedback("commands.ckit.create.success", name);
+        return 0;
+    }
+
+    private static int delete(ServerCommandSource source, String name) throws CommandSyntaxException {
+        loadFile();
+
+        if (kits.remove(name) == null) {
+            throw NOT_FOUND_EXCEPTION.create(name);
+        }
+        saveFile();
+        sendFeedback("commands.ckit.delete.success", name);
+        return 0;
+    }
+
+    private static int edit(ServerCommandSource source, String name) throws CommandSyntaxException {
+        loadFile();
+
+        if (kits.get(name) == null) {
+            throw NOT_FOUND_EXCEPTION.create(name);
+        }
+        kits.put(name, client.player.inventory.serialize(new ListTag()));
+        saveFile();
+        sendFeedback("commands.ckit.edit.success", name);
+        return 0;
+    }
+
+    private static int load(ServerCommandSource source, String name, boolean override) throws CommandSyntaxException {
+        if (!client.player.isCreative() || !client.player.abilities.creativeMode) {
+            throw NOT_CREATIVE_EXCEPTION.create();
+        }
+
+        loadFile();
+
+        ListTag kit = kits.get(name);
+        if (kit == null) {
+            throw NOT_FOUND_EXCEPTION.create(name);
+        }
+
+        PlayerInventory tempInv = new PlayerInventory(client.player);
+        tempInv.deserialize(kit);
+        List<Slot> slots = client.player.playerScreenHandler.slots;
+        for (int i = 0; i < slots.size(); i++) {
+            if (slots.get(i).inventory == client.player.inventory) {
+                ItemStack itemStack = tempInv.getStack(((ISlot) slots.get(i)).getIndex());
+                if (!itemStack.isEmpty() || override) { // same as if (!(itemStack.isEmpty() && !override))
```

Comment:
Unnecessary comment here

Reply:
I chose to add that command because `if (!itemStack.isEmpty() || override)` might seem totally arbritary to anyone who doesn't know what is being checked here

Reply:
It's quite obvious, we all know de-morgan's law

Reply:
The what?! I have no idea what that is 🤣